lib/mergometer.rb

```diff
@@ -1,21 +1,10 @@
 require "facets/math"
 require "octokit"
+require "ostruct"
```

Comment:
Where is this used? I'd recommend to include it from the file that requires it, instead of globally.